docker/lighthouse-server/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /et
     && chmod +x /usr/bin/lh
 
 # Add the executable.
-ADD bin/lh-server /
+ADD bin/lighthouse-server /
```

Comment:
This is controlled by an ENV var everywhere else.... should it be done here too?

Reply:
This is the name of the binary that gets built and added to the image, it shouldn't be configurable IMO

Reply:
works for me